Policies/Subscription/Azure_Subscription_Config_MDC_Setup_SecurityContacts/Readme.md

```diff
@@ -0,0 +1,24 @@
+## Azure_Subscription_Config_MDC_Setup_SecurityContacts
```

Comment:
We should also add this control and link to policy page in policy index/readme.md file as well.

Reply:
Added